conf/environments.js

```diff
@@ -33,5 +33,17 @@ module.exports = {
     },
     jasmine: {
         globals: globals.jasmine
+    },
+    phantom: {
+        globals: globals.phantom
+    },
```

Comment:
@nzakas @sindresorhus I suppose,
I will wait for you final verdict and apply changes